pkg/detectors/salesmate/salesmate.go

```diff
@@ -44,24 +44,23 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 		resMatch := strings.TrimSpace(match[1])
 		for _, idmatch := range idmatches {
-			if len(idmatch) != 2 {
```

Comment:
@ankushgoel27 this shouldn't be removed

Reply:
if i dont remove this, the detector fails